backend/tests/test_progress.py

```
@@ -163,6 +164,83 @@ def progress_callback(downloaded: int, total: int, filename: str):
+
+                with tqdm(total=size, desc=filename, unit="B") as pbar:
+                    for _ in range(0, size, chunk):
+                        pbar.update(chunk)
+                        time.sleep(0.005)
+        except Exception as e:  # pragma: no cover - surfaced via `errors`
+            errors.append(e)
+
+    # HFProgressTracker only reports progress once a file's total crosses
+    # MIN_TOTAL_BYTES (1MB) — use real-sized files so this test exercises
+    # the same path a real download does.
+    SIZE_A = 2_000_000
```

Ruff (0.15.20)

[warning] Variable `SIZE_A` in function should be lowercase

(N806)